pkg/alpine/mirrors/3.16/community

```diff
@@ -36,3 +36,13 @@ tpm2-tss-tctildr
 tpm2-tss-fapi
 tpm2-tss-rc
 pkgconf
+librados
```

Comment:
I think you need to update pkg/alpine and merge that so that you get a new tag for eve-alpine in docker hub, then use that tag in pkg/*/Dockerfile. Please check with Rene/Roman who has done this recently AFAIR.

Reply:
Done